CHANGELOG.md

```diff
@@ -1,40 +1,49 @@
 # Changelog
 
-All notable changes to this project will be documented in this file.
-
-The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
-and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+All notable changes to this project will be documented in this file. The project adheres to
+  [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
```

Comment:
nitpick: why there're two spaces in the beginning of the string?

Reply:
There are two spaces at the beginning of other lines. As for me, it is easier to review especially with lists.

Reply:
Yeah, but this isn't a list... I fully support using indentation for lists (some Markdown parsers won't even understand that something is a list otherwise), but for plain paragraph text it seems a bit strange.

Reply:
I prefer unification whenever is possible. :smile: Simple rule "use two spaces if you want to split a line" seems nice, but I'm going to remove one space if you are insist.

Reply:
Now I don't get it. What line split are you talking about? There is no line break before "semantic versioning" in [the rendered file](https://github.com/DarkEld3r/exonum/blob/a679fce7dfa548d3a9c913319c0eb540a98bd268/CHANGELOG.md), neither it should be arguably. There is no need to have spaces at the beginning of a line in the paragraph text at all.